src/components/Auth/LoginScreen.js

```diff
@@ -18,35 +19,47 @@ const LoginScreen = ({ navigation }) => {
   const [token, setToken] = useState("");
   const [password, setPassword] = useState("");
 
-  const connect =  () => {
-
-    fetch('http://192.168.0.13:5000/auth/login', {
-      method: 'post',
-
-      headers: {
-        Accept: 'application/json',
-        'Content-Type': 'application/json'
-      },
 
-      body: JSON.stringify({
-        email: email,
-        password: password
-      })
-      })
-      .then((res)=>{
-        return res.text()
-      })
+  const connect =  () => {
+    axios.post('http://192.168.0.13:5000/auth/login',{
```

Comment:
Probably should change this to `localhost` since the ip could potentially change